ci_framework/roles/build_openstack_packages/molecule/default/prepare.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Prepare
+  hosts: all
+  vars:
+    cifmw_basedir: "/tmp/ci-framework"
```

Comment:
ditto - `{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework`

note that you may need to add this as well:
`ansible_user_dir: "{{ lookup('env', 'HOME') }}"`